src/MoBi.Presentation/Presenter/EditSimulationPresenter.cs

```diff
@@ -216,6 +223,8 @@ public void Handle(SimulationRunFinishedEvent eventToHandle)
          if (!_simulation.Equals(eventToHandle.Simulation))
             return;
 
+         _view.SetParametersTabEnabled(true);
+
```

Comment:
why? if we disable parameters tab during a simulation run and then add obsdata while sim is till running, it will be enabled again(?)

Reply:
yes Idon't get this either